src/sedtrails/configuration_interface/configuration_controller.py

```diff
@@ -57,11 +55,16 @@ def read_config(self, config_file: str) -> None:
             The path to the configuration file to read.
         """
 
-        #TODO: Implement reading from a configuration file
-        # read the configuration file
-        # validate the configuration
+        valid_json = os.path.join(r'/config_schema.json')  # not happy with hardcoded path
```

Comment:
The way I understand it is that the `config_schema.json` is part part of the source code; therefore, there is no problem with hard-coding the path.